### ISSUE TITLE
Obsidian Registration Entry Will NOT Show Form when Allow Multiple Registrants is Not Checked

### ISSUE THREAD

**Issue by frogydiak** [SparkDevNetwork/Rock#5538]
### Please go through all the tasks below

- [X] Check this box only after you have successfully completed both the above tasks

### Please provide a brief description of the problem. Please do not forget to attach the relevant screenshots from your side.

The Obsidian registration entry form does appear in the page and I'm getting a console error:

![image](https://github.com/SparkDevNetwork/Rock/assets/1373815/cfe90123-29c5-4572-8970-edcac185f9f6)

### Expected Behavior

The form should appear even with single registrant. Unless, there's a reason for it, may I know why?

![image](https://github.com/SparkDevNetwork/Rock/assets/1373815/853abfd0-4255-4691-b2f6-f04c31e04ffa)

### Actual Behavior

See problem description.

If I enable the Allow Multiple Registrants, the form works perfectly fine:

![image](https://github.com/SparkDevNetwork/Rock/assets/1373815/9829dce5-5353-402e-8bb9-079528c947d7)

![image](https://github.com/SparkDevNetwork/Rock/assets/1373815/ab0639fa-05dd-4007-a786-cfa79f7d96bd)

### Steps to Reproduce

- Uncheck the Allow Multiple Registrants

### Rock Version

15.1

### Client Culture Setting

en-US

**Comment by frogydiak** [comment]
> I am unable to reproduce this on the rock demo site. I have a few questions.
> 
> 1. Do you have anything in pre or post html on one of your fields?
> 2. Is this the obsidian registration entry block or the old one?
> 3. Are there any other settings you changed from the default new registration settings?

Hi @robotman3000! 

1. Yes, I do have in number of fields like Mobile Phone, Email, etc. but the only HTML I use is to created a 2 column bootstrap grid.
2. It is Obsidian registration entry block.
3. I use the default settings except for the block title:
![image](https://github.com/SparkDevNetwork/Rock/assets/1373815/8f009490-dc6f-474d-8e64-2cec88cd35bb)

**Comment by robotman3000** [comment]
I am tracking down another issue that has the same symptoms. But the conditions required for it to occur don't seem to be met here. Can you verify that your pre/post html isn't missing any tags and nothing is mistyped?

**Comment by frogydiak** [comment]
@robotman3000 Yes, you are right! I have a post html in one of the field that I mistyped the closing tag. 

Maybe it would help people if the obsidian form raise an error or warning with html issue instead of not showing anything at all.

Anyhow, thank you so much! I will close this issue.

**Comment by Jon-Corey** [comment]
Hey @frogydiak, the Obsidian block does actually throw an error when there is an error with the HTML. If there is an error with the HTML, a `SyntaxError` will be logged to the browser's console. The error isn't very descriptive, but it does let you know that the issue is likely with the HTML.

Here's a resource that describes how to open your browser's console: https://support.monday.com/hc/en-us/articles/360002197259-How-to-open-the-developer-console

**Comment by frogydiak** [comment]
Hi @Jon-Corey, well, yeah but that is if you know where and how to look. I don't know how much computation is needed to make that error appear in the UI while editing. In my opinion showing the issue when saving would make Rock's UI better (and a better product) rather the user has to dig that issue in the developer console and wonder "where did I go wrong?".
